docs/monitoring/tags.md

```diff
@@ -0,0 +1,12 @@
+title: Tags 
+description: A Tag is an attribute of a data point (or metric) that could be used for grouping and filtering.
+
+[Tags](../../tags) are sent by Sematext's Agents as part of every data point. Tags are shown in the UI as filters and group by elements. 
+
+For example, as part of Docker container metrics, the agent sends hostname on which the container is running, container identifier, container name, container image as tags which appear as filters in UI. User can then group or filter the container metrics using these tags.
+
+All Tags from the [Common Schema](../../tags/common-schema) apply to both Monitoring and Logs Apps.
```

Comment:
Well,  not really ALL tags.  We have them grouped there.  So maybe we can just drop "All".
Does Experience have any "reserved" tags?  Or SM?  Could you please find out and add them if appropriate?  Or ask DRIs to send a PR?

Reply:
Fixed.

I've asked the boys, I'll get an answer tomorrow.

Reply:
Amir said Experience does not have any reserved tags. Siva said he'll check and get back to me tomorrow.

Reply:
I've added tags for SM in the Common Schema.